CONTRIBUTING.md

```diff
@@ -52,11 +52,7 @@ Algorithms in this repo should not be how-to examples for existing JavaScript pa
 #### Coding Style
 
 To maximize the readability and correctness of our code, we require that new submissions follow [JavaScript Standard Style](https://standardjs.com/)
-  - Command to  install JavaScript Standard Style
-    ```
-    $ npm install standard --save-dev
-    ```
-  - Usage
+  - Usage of JavaScript Standard Style
```

Comment:
hello @Swapnil-2001 please, explain your changes

Reply:
The **standard** module is already included, so I was wondering if we can do without installing it everytime we make changes.

Reply:
yes, it is but CONTRIBUTING.md  shows the use  cases and how to run modules  
its an outline template and guidelines to  contribute

Reply:
so it just tells if you want to install standard user can use this command

but it could be better if we use `npx` in place of `npm`
@Swapnil-2001 can use replace npm command with `npx ` one
so if anyone want to test a file without installing(npm install) they can do  that

Reply:
Okay, my apologies, I didn't understand that bit.